src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Builds.cs

```diff
@@ -0,0 +1,95 @@
+using System.Collections.Immutable;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Maestro.Client.Models;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Maestro.ScenarioTests
+{
+    [TestFixture]
+    public class ScenarioTests_Builds : MaestroScenarioTestBase
+    {
+        private string repoUrl;
+        private readonly string repoName = "maestro-test1";
+        private readonly string sourceBuildNumber = "654321";
+        private readonly string sourceCommit = "123456";
+        private readonly string sourceBranch = "master";
+        private readonly IImmutableList<AssetData> sourceAssets = ImmutableList<AssetData>.Empty;
+
+        public ScenarioTests_Builds()
+        {
+            sourceAssets = GetAssetData("Foo", "1.1.0", "Bar", "2.1.0");
+        }
+
+        // Create a new build and check some of the metadata. Then mark as released and check again
+        [Test]
+        public async Task ArcadeBuilds_EndToEnd()
+        {
+            TestContext.WriteLine("Darc/Maestro build handing tests");
+
+            repoUrl = GetRepoUrl(repoName);
+
+            // Create a build for the source repo
+            Build build = await CreateBuildAsync(repoUrl, sourceBranch, sourceCommit, sourceBuildNumber, sourceAssets);
+            Build retrievedBuild = await MaestroApi.Builds.GetBuildAsync(build.Id);
+            Assert.IsFalse(retrievedBuild.Released);
+
+            //Release the build
+            Build updatedBuild = await MaestroApi.Builds.UpdateAsync(new BuildUpdate() { Released = true }, build.Id);
+            Assert.IsTrue(updatedBuild.Released);
+
+            // Gather a drop with release included
+            string gatherWithReleasedDir = Path.Combine(base._parameters._dir + "gather-with-released");
```

Comment:
the "base" here isn't necessary, and our style generally avoids it unless it's required (to call a base method that's been overridden is the only time that's required)